### ISSUE TITLE
Search bar: Cache and bookmarking

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#62]
## User story
1. As a User
2. I want see previous searches and be able to bookmark them
3. So that I dont have to re-enter the search term again

## Acceptance criteria
* Display list of previous searches
* Ability to bookmark them for the current session, so they appear different in the search list

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by op-hub** [comment]
Also already implemented: remove Duplicate
